bc/core/utils/network.py

```diff
@@ -31,3 +31,10 @@ def strip_port_to_make_ip_key(group: str, request: HttpRequest) -> str:
     method=ratelimit.UNSAFE,
     block=True,
 )
+
+ratelimiter_unsafe_1_per_5m = ratelimit(
+    key=strip_port_to_make_ip_key,
+    rate="1/5m",
```

Comment:
This seems pretty aggressive. Like, I could see myself being annoyed with it, if I had a handful of ideas to input. Maybe 5/30m would be better?